vllm/envs.py

```diff
@@ -93,6 +93,14 @@
     VLLM_FORCE_AOT_LOAD: bool = False
     VLLM_TORCH_PROFILER_WITH_STACK: bool = True
     VLLM_TORCH_PROFILER_WITH_FLOPS: bool = False
+    USE_PROTON: bool = False
+    PROTON_PROFILE_NAME: str | None = None
```

Comment:
This seems a lot, wondering if we can infer USE_PROTON from PROTON_PROFILE_NAME, also for the ENV vars, we can try to infer as many as possible.